invision-app/app/components/movieCard/movieCard.js

```diff
@@ -1,92 +1,88 @@
-import React from 'react';
+import React,{useState} from 'react';
 import './movieCard.scss';
 import { Menu } from "../menu/menu";
 import { AddEditMovie } from "../addEditMovie";
 import { DeleteMovie } from "../deleteMovie/deleteMovie";
-export class MovieCard extends React.Component {
+import PropTypes from 'prop-types';
 
-  constructor(props) {
+export function MovieCard(props) {
+  const [isHovered, setHoverState] = useState(false);
+  const [isOpenedMenu, setMenuState] = useState(false);
+  const [isOpenedDeleteWindow, setDeleteState] = useState(false);
+  const [isopenedEditWindow, setAddEditState] = useState(false);
```

Comment:
**setAddEditState** is not understandable what exactly this function should do. You can name this function like **setVisibilityOfEditWindow**. The same thing wuth **setMenuState**

Reply:
Modified.